content/en/platform/corda/4.9/enterprise/release-notes-enterprise.md

```diff
@@ -34,6 +34,10 @@ As a developer or node operator, you should upgrade to the [latest released vers
 
 Corda Enterprise Edition 4.9.4 is a patch release of Corda Enterprise focused on resolving issues.
 
+{{< note >}}
+ The Corda Shell has a dependency on Groovy which is not on the public API. With the removal of the shell the groovy dependency is also removed. Any flows dependent on groovy need to add the dependency to the CorDapp, the drivers directory, or remove the use of Groovy.
```

Comment:
This isn't very clear to me. What has changed in this release? Is it that the shell now has a dependency on Groovy? Or that the API no longer has a dependency? Or that the shell is removed?

Reply:
In 4.9 the shell was removed from the product and made as a stand alone product. However the shell had a dependency on Groovy which was never on the public API. As some customers rely on Groovy in their flows the note was added to mention that along with the shell, groovy should also be removed!

Reply:
Ok, so it's not specific to 4.9.4 so probably shouldn't be in that section? Maybe check with the reporter but it probably doesn't make sense to retrospectively add it to the 4.9 section. Since it's a note, you could probably put it at the top of the page maybe?

I'd also suggest rewording it a little to make the note clearer. Something like:

```The Corda Shell, which has a dependency on Groovy, is now a standalone product since Corda Enterprise Edition 4.9. As a result, for any flows dependent on Groovy, you must add the dependency to the CorDapp or the drivers' directory.
```